includes/class-sensei-usage-tracking-data.php

```diff
@@ -51,7 +52,111 @@ public static function get_usage_data() {
 			'teachers'                => self::get_teacher_count(),
 		);
 
-		return array_merge( $question_type_count, $usage_data );
+		return array_merge( $question_type_count, $usage_data, $quiz_stats );
+	}
+
+	/**
+	 * Get stats related to lesson quizzes.
+	 *
+	 * @since 1.11.0
+	 *
+	 * @return array
+	 */
+	private static function get_quiz_stats() {
+		$query = new WP_Query( array(
+			'post_type'      => 'lesson',
+			'fields'         => 'ids',
+			'posts_per_page' => -1,
+			'no_found_rows'  => true,
+			'meta_query'     => array(
+				array(
+					'key'   => '_quiz_has_questions',
+					'value' => true,
+				),
+				array(
+					'key'     => '_lesson_course',
+					'value'   => '',
+					'compare' => '!=',
+				),
+				array(
+					'key'     => '_lesson_course',
+					'value'   => '0',
+					'compare' => '!=',
+				),
+			),
+		) );
+
+		$stats = array(
+			'quiz_total'         => 0,
+			'questions_min'      => null,
+			'questions_max'      => null,
+			'category_questions' => 0,
+		);
+		$question_counts    = array();
+		$published_quiz_ids = array();
+
+		foreach ( $query->posts as $lesson_id ) {
+			$course_id = Sensei()->lesson->get_course_id( $lesson_id );
+			if ( empty( $course_id ) || 'publish' !== get_post_status( $lesson_id ) || 'publish' !== get_post_status( $course_id ) ) {
+				continue;
+			}
+
+			$quiz_id              = Sensei()->lesson->lesson_quizzes( $lesson_id );
+			$quiz_question_posts  = Sensei()->lesson->lesson_quiz_questions( $quiz_id );
+			$question_counts[]    = count( $quiz_question_posts );
+			$published_quiz_ids[] = $quiz_id;
+			$stats['quiz_total']++;
+		}
+
+		if ( ! empty( $published_quiz_ids ) ) {
+			$multiple_question_query     = new WP_Query( array(
+				'post_type'        => 'multiple_question',
+				'posts_per_page'   => -1,
+				'fields'           => 'ids',
+				'no_found_rows'    => true,
+				'suppress_filters' => 1,
+				'meta_query'       => array(
+					array(
+						'key'   => '_quiz_id',
+						'value' => $published_quiz_ids,
+					),
+				),
+			) );
+			$stats['category_questions'] = self::get_category_question_count( $published_quiz_ids );
+		}
+
+		if ( ! empty( $question_counts ) ) {
+			$stats['questions_min'] = min( $question_counts );
+			$stats['questions_max'] = max( $question_counts );
+		}
+
+		return $stats;
+	}
+
+	/**
+	 * Get the number of category/multiple questions assigned to published quizzes.
+	 *
+	 * @since 1.11.0
+	 * 
+	 * @param int[] $published_quiz_ids
+	 * @return int
+	 */
+	private static function get_category_question_count( $published_quiz_ids ) {
+		$multiple_question_query     = new WP_Query( array(
```

Comment:
When you add multiple questions as a "Category Question", should the count follow the total number of questions that have been added, or the number of "multi-question categories" added?

When I add a category now, with 2 questions, it only counts for 1 in the `category_questions` key being sent to Tracks.

On the other hand, it does count for multiple questions in the total count. I.e. if my largest quiz has 5 questions, and 1 category question that includes 2 questions in the quiz, then my `questions_max` is 7.

Is this the intention? (I'm not saying it shouldn't be, I actually think it makes sense, but I'm just making sure 😄 )

Reply:
I interpreted @donnapep's issue as counting the number of times a category question is added to a quiz, not how many questions is pulled when doing so. However, we can change this around if that makes more sense. 

`questions_min` and `questions_max` gets the fully expanded question set, with the questions pulled from the category. I think this makes sense.